src/features/surveys/components/EditWarningCard.tsx

```diff
@@ -68,14 +67,16 @@ const EditTips: FC<{
 }> = ({ bullets, header, icon }) => {
   return (
     <Box display="flex" gap={2} marginTop={1}>
-      <Box>{icon}</Box>
       <Box>
-        <Typography fontWeight="bold">{header}</Typography>
-        <ul style={{ marginTop: 0, paddingLeft: 16 }}>
+        <Typography>{header}</Typography>
+        <Typography style={{ marginTop: 2 }}>
```

Comment:
I think you should change this Typography to be a Box instead, because it has children. A Typography/`p` should not have children that are Box/`div`